### PR TITLE
Updating the gh-pages with Get started page.

### LINKED ISSUE #70: Make bar charts samples to start from "1" interred of 0
**Description:**
Currently, representation with 0 looks confusing and It will not be used. 
Also, have a sample with some text in x-axis.

### LINKED ISSUE #71: no link to source code from the github io site.
**Description:**
Have a link to source code from io site.

### LINKED ISSUE #72: Inline charts are not drawing correctly.
**Description:**
$Subject & the name is also wrong. 

screen shot 2018-02-05 at 12 47 33 pm

### LINKED ISSUE #73: Make all tooltip font and color the same
**Description:**
$Subject for map and others

### PR DESCRIPTION
**Pull request by CharukaK**
## Purpose
resolves #73 
resolves #72 
resolves #71 
resolves #70 

## Test environment
Node.JS v8.8.1, NPM v5.4.2

### CODE REVIEW

File: dist/components/resources/themes/victoryDarkTheme.js [hunk 1]
```diff
@@ -0,0 +1,228 @@
+'use strict';
```
Comment (lasanthaS) [inline dist/components/resources/themes/victoryDarkTheme.js]: Add license header

File: dist/components/resources/themes/victoryLightTheme.js [hunk 2]
```diff
@@ -0,0 +1,228 @@
+'use strict';
```
Comment (lasanthaS) [inline dist/components/resources/themes/victoryLightTheme.js]: Add license header

File: dist/components/resources/themes/victoryDarkTheme.js (hunk 1 above)
Comment (CharukaK) [reply dist/components/resources/themes/victoryDarkTheme.js]: done

File: dist/components/resources/themes/victoryLightTheme.js (hunk 2 above)
Comment (CharukaK) [reply dist/components/resources/themes/victoryLightTheme.js]: done

File: dist/components/resources/themes/victoryDarkTheme.js (hunk 1 above)
Comment (CharukaK) [reply dist/components/resources/themes/victoryDarkTheme.js]: done

**Pull request merged**